starter-code/views/not-found.ejs

```diff
@@ -0,0 +1 @@
+<p> The requested URL was not found on this server </p>
```

Comment:
Instead of having a file for each error you can have a template like this for all the errors. You just have to render the page and pass an error json
```
<h1><%= message %></h1>
<h2><%= error.status %></h2>
<pre><%= error.stack %></pre>
```